example/testapp/models.py

```diff
@@ -9,10 +9,13 @@ class Car(models.Model):
     def __unicode__(self):
         return self.plate
 
```

Comment:
Shall I use tools like [autopep8](https://pypi.python.org/pypi/autopep8) to correct all pep8 errors?